utils/build/build.js

```diff
@@ -130,7 +133,16 @@ class ProgramStep extends Step {
       },
       cwd: step.cwd,
     });
-    process.on('exit', () => child.kill());
+
+    if (ProgramStep.activeChildren.length === 0) {
```

Comment:
While this works, I think it's nicer to have a generic list of disposables to run at exit, sigint, etc. We can probably include esbuild contexts there, and calls `context.dispose()` to ensure esbuild native process is stopped.